indico/cli/event.py

```diff
@@ -85,3 +86,40 @@ def import_(source_file, create_users, force, verbose, yes, category_id):
         sys.exit(1)
     db.session.commit()
     click.secho(event.external_url, fg='green', bold=True)
+
+
+@cli.command('create-series')
+@click.option('--title-sequence/--no-title-sequence', 'show_sequence_in_title', default=True,
+              help='Whether to show the series sequence in the event titles (lectures only)')
+@click.option('--links/--no-links', 'show_links', default=True,
+              help='Whether to show links to other events in the series on the event page')
+@click.argument('event_ids', nargs=-1, type=int, metavar='EVENT_ID...')
+def create_series(show_sequence_in_title, show_links, event_ids):
+    """Create a series from a list of events."""
+    events = Event.query.filter(Event.id.in_(event_ids), ~Event.is_deleted).all()
+
+    if missing := set(event_ids) - {e.id for e in events}:
+        click.secho('Event not found:', fg='red')
+        for event_id in missing:
+            click.echo(f'- {event_id}')
+        sys.exit(1)
+    elif assigned := [e for e in events if e.series]:
+        click.secho('Event already assigned to a series:', fg='red')
+        for event in assigned:
+            click.echo(format_event(event))
+        sys.exit(1)
```

Comment:
I'd simplify the missing events a bit - `', '.join(str(id) for id in missing)` instead of the loop - but nothing wrong with printing them in individual lines, so feel free to leave it as it is.

So yes, it looks pretty good and it's what I meant.

Just a small thing I'd change:

```diff
-    if missing := set(event_ids) - {e.id for e in events}:
+     if missing := (set(event_ids) - {e.id for e in events}):
```

it just makes it clearer what's being assigned

(and maybe rename `assigned` to `conflict` in the elif?)

Reply:
I'm in favor of printing one event per line in this instance. It prints like this in the other cases as well so there is
some uniformity. I renamed the variable to `conflict`. I also considered `taken` but then I went for `assigned` since that is also used in the cli output. Although, now I think `conflict` is the most explicit.